packages/codebytes/package.json

```
@@ -25,7 +25,11 @@
     "@codecademy/gamut-styles": "*",
     "@codecademy/variance": "*",
     "@emotion/react": "^11.4.0",
-    "@emotion/styled": "^11.3.0"
+    "@emotion/styled": "^11.3.0",
+    "@loadable/component": "^5.13.1",
+    "react-monaco-editor": "0.34.0",
```

```suggestion
    "@monaco-editor/react": "4.3.1",
```

thoughts on switching this library with [@monaco-editor/react](https://www.npmjs.com/package/@monaco-editor/react)? - it will remove the need for webpack configuration in repos that use this package

---

like it! since i haven't tested functionality with that package, i'll just confirm that it's working as expected today.